django/src/rdwatch/models/site.py

```diff
@@ -15,6 +19,8 @@ class Site(models.Model):
     )
     saliency = models.ForeignKey(
         to="Saliency",
+        null=True,
+        blank=True,
```

Comment:
Is this related to the [note here](https://github.com/ResonantGeoData/RD-WATCH/blob/98b94a11303be53937a417e14e3e780fc2f79789/django/src/rdwatch/management/commands/ingest.py#L176-L179)? Until we know for sure what's going on there, we should assume we're missing data rather than loosening our constraints. Right now, I can't mentally reconcile how a `Site` not derived from a `Saliency` can exist and our data model should reflect that.